packages/@expo/cli/src/export/exportDomComponents.ts

```diff
@@ -98,64 +108,63 @@ export async function exportDomComponentAsync({
   };
 }
 
+//#region `npx export` for updates
+
+/**
+ * Add the DOM component bundle to the metadata for updates.
+ */
+export function addDomBundleToMetadataAsync(bundle: BundleOutput): PlatformMetadata['assets'] {
+  const assetsMetadata: PlatformMetadata['assets'] = [];
+  for (const artifact of bundle.artifacts) {
+    if (artifact.type === 'map') {
+      continue;
+    }
+    assetsMetadata.push({
+      path: `${DOM_COMPONENTS_BUNDLE_DIR}/${artifact.filename}`,
```

Comment:
Do the assets need to have md5 hash names? Seems like maybe they were just missing from the metadata.json file?

Reply:
it's not necessary actually. expo-updates will get the metadata but persist files in flattened md5 structure. but i feel it would be good for cli export to have aligned resolution. that would help test predictable.